airflow/operators/python.py

```diff
@@ -408,7 +408,8 @@ def __init__(
         if not requirements:
             self.requirements: Union[List[str], str] = []
         elif isinstance(requirements, str):
-            self.requirements = requirements
+            with open(requirements, "r") as file:
+                self.requirements = file.readlines()
```

Comment:
This does not work if the task is distributed onto a worker, which is not necessarily the same machine as the scheduler. The file should only need to exist on the worker (that only runs the task code), not the scheduler (which runs the Airflow scheduling logic).

So this must continue to store only the filename in `__init__`, and reading the file can only happen in `execute`.

Reply:
Overriding `prepare_template` to post-process `self.requirements` may be a viable approach to achieve this.

Reply:
Thanks, I was not aware of that. I have moved the reading of the file to `execute_callable()`